apps/system-e2e/src/tests/judicial-system/smoke/custody-tests.spec.ts

```diff
@@ -220,7 +220,10 @@ test.describe.serial('Custody tests', () => {
 
     // Confirmation
     await expect(page).toHaveURL(`/domur/stadfesta/${caseId}`)
-    await page.getByTestId('continueButton').click()
+    await Promise.all([
+      await page.getByTestId('continueButton').click(),
```

Comment:
No need for await if you're awaiting the collector
```suggestion
      page.getByTestId('continueButton').click(),
```